ql/src/java/org/apache/hadoop/hive/ql/optimizer/ConvertJoinMapJoin.java

```diff
@@ -857,6 +865,26 @@ private boolean checkConvertJoinSMBJoin(JoinOperator joinOp, OptimizeTezProcCont
     return true;
   }
 
+  private boolean hasMoreGBYs(Operator<?> start, int max) {
```

Comment:
A brief comment for this method would be good.

Reply:
Added.